public/doc/swagger-2.yaml

```diff
@@ -861,6 +869,22 @@ definitions:
         type: string
         description: Associated template id
         readOnly: true
+  WorkflowGroups:
+    type: object
+    description: The associated approval groups
```

Comment:
`description: Parameters to create a workflow`

Reply:
@bzwei Changed based on your comments.